cmd/snap-exec/main.go

```diff
@@ -67,6 +67,8 @@ func run() error {
 func findCommand(app *snap.AppInfo, command string) (string, error) {
 	var cmd string
 	switch command {
+	case "debug-shell":
```

Comment:
s/debug-//.. as we discussed online, it seems fine to have it as `--shell` alone assuming it is under the `run` command. Let's please keep the `snap shell` command to that other feature, though.